src/main/resources/META-INF/plugin.xml

```diff
@@ -72,14 +72,14 @@
   <depends>com.intellij.modules.lang</depends>
   <depends>org.jetbrains.plugins.terminal</depends>
   <depends>com.redhat.devtools.intellij.telemetry</depends>
+  <depends>com.redhat.devtools.intellij.kubernetes</depends>
 
   <depends optional="true" config-file="withJava.xml">com.intellij.modules.java</depends>
   <depends optional="true" config-file="withJavaScriptDebugger.xml">JavaScriptDebugger</depends>
   <depends optional="true" config-file="plugin-json.xml">com.intellij.modules.json</depends>
 
   <extensions defaultExtensionNs="com.intellij">
-    <!-- Add your extensions here -->
-    <toolWindow id="OpenShift" anchor="left" factoryClass="org.jboss.tools.intellij.openshift.WindowToolFactory" icon="/META-INF/pluginIcon13x13.svg"/>
+    <toolWindow id="OpenShift" anchor="left" factoryClass="org.jboss.tools.intellij.openshift.WindowToolFactory" icon="/META-INF/OpenShiftPluginIcon13x13.svg"/>
```

Comment:
Not sure this renaming is worth it

Reply:
I had issues with the icons of the views when running my dev instance. both kubernetes and openshift had the same icon ( kube ). renaming it fixed it.